lib/ioki/model/operator/vehicle.rb

```diff
@@ -44,6 +44,10 @@ class Vehicle < Base
                   omit_if_nil_on: [:create, :update],
                   type:           :string
 
+        attribute :fuel_type_options,
+                  on:   :read,
+                  type: :array
```

Comment:
Does this have a better type that we can describe with `classname` or is this just an arbitrary array of strings?

Reply:
The reason for adding this is to make sure that all fuel type options are included in a dropdown selection for the vehicle creation.

Reply:
Thanks for clarifying!